frontend/src/components/Login.js

```diff
@@ -1,10 +1,110 @@
 import React from 'react';
+import {useState} from 'react'
+import axios from 'axios';
+import jwt_decode from "jwt-decode";
+import {TextField, Button, Typography, Grid, DialogContentText, Link} from '@mui/material';
+import SideBar from "./SideBar";
+import TopBar from "./TopBar";
+import StickyFooter from "./Footer";
 
-const Login = () => {
-    return (
-        <div>
-        </div>
-    )
+const Login = ({userInformations, setUserInformations}) => {
+    const [errorMessage, setErrorMessage] = useState()
+
+    const logUserIn = (evt) => {
+        // Prevents the form from being submitted
+        evt.nativeEvent.preventDefault();
+
+        // Get the form fields
+        const formElements = evt.nativeEvent.target.elements;
+        const email = formElements.email.value;
+        const password = formElements.password.value;
+
+        axios ({
+            method: 'POST',
+            baseURL: 'http://localhost:8080',
+            url: '/user/login',
+            data: JSON.stringify({ email: email, password: password }),
+            headers: {
+                'content-type': 'application/json',
+            },
+          })
+          .then(function (response) {
+            if (response.data !== ''){
+                sessionStorage.setItem('jwt', `Bearer ${response.data}`);
+
+                const decodedJWT = jwt_decode(response.data);
+                setUserInformations({
+                    email: email,
+                    role: decodedJWT.role,
+                    loggedIn: true
+                });
+                setErrorMessage();
+              } else {
+                let errorMessage = 'Informations ne correspondent à aucun utilisateur sauvegardé'
+                setErrorMessage(errorMessage);
+              }
+        })
+        .catch(function (error) {
+            let errorMessage = 'Informations ne correspondent à aucun utilisateur sauvegardé'
```

Comment:
Message should reflect what happened: Please use something along the lines of "unexpected error, server error) (in french of course)

Reply:
The error caught by the post request is a 401 Unauthorized, which is sent by the backend when the given credentials do not match a saved user, justifying the message showed to the user if an exception is caught.